packages/truffle-core/lib/build.js

```diff
@@ -0,0 +1,133 @@
+const async = require("async");
+const mkdirp = require("mkdirp");
+const del = require("del");
+const fs = require("fs");
+const Contracts = require("truffle-workflow-compile");
```

Comment:
is there a reason the Contracts and BuildError constants are capitalized in the first letter or should they start with a lowercase?

Reply:
That's just convention. Typically it depends what gets returned... if you're requiring a class or something that looks like a class, UpperCamelCase seems appropriate

Reply:
Ah, that makes sense. Will keep that in mind, thanks!